submissions/phemmelliot/Week1/Dufuna.txt

```diff
@@ -0,0 +1,25 @@
+package edu.princeton.cs.algs4;
+
+/**
+ * Created by phemmelliot on 4/24/18.
+ */
+public class Dufuna {
+    public static void main(String[] args){
+        for(int i = 1; i<=100; i++){
+            check(i);
+        }
+    }
+    // i used this algorithm since no optimization will change the runtime significantly
+    // O(N) and O(logN) won't be different here for both time and space complexity
```

Comment:
This comment is very confusing. I understand you meant the best time complexity we can have here is O(N), since we have to touch all numbers between 1 - 100. But I don't get why you mentioned O(logN). In fact, one can argue this program is O(1) because N in this case is always 100 :)